### DISCUSSION TITLE
antd v5 - Form scrollToFirstError

### DISCUSSION THREAD

**Discussion by erdemkeren** [alibaba/formily#3983] [Q&A 2.x]
I am looking for the equivalent of the scrollToFirstError functionality of antd forms that can be found here

https://ant.design/components/form#api
https://github.com/scroll-into-view/scroll-into-view-if-needed/tree/ece40bd9143f48caf4b99503425ecb16b0ad8249#options

Can we use it or is there an equivalent that I missed?

**Comment by sTeADone** [comment]
You can try something like
```
document.querySelector(".ant-formily-item-error").scrollIntoView()
```
in `Submit`'s `onSubmitFailed` property
as memtioned in https://github.com/alibaba/formily/issues/3261#issuecomment-1241623979
